app/routes/notifications.js

```diff
@@ -2,41 +2,64 @@
 
 const express = require('express');
 const router = express.Router(); // eslint-disable-line new-cap
+const helpers = require('../../lib/helpers');
 const mobilecommons = rootRequire('lib/mobilecommons');
 const logger = app.locals.logger;
 const stathat = app.locals.stathat;
 
+const allowedReminderTypes = ['signup', 'reportback'];
+
 router.post('/reminder', (req, res) => {
```

Comment:
@sergii-tkachenko Were you going to change this `/notifications/reminder` endpoint, or is that for a different PR?